src/app.py

```diff
@@ -65,6 +65,12 @@ def bootstrap(_return=0):
         try:
             master_host = get_or_raise('MASTER_HOST')
             master_url = 'http://{master}:8089'.format(master=master_host)
+            total_slaves = int(os.getenv('TOTAL_SLAVES')) if os.getenv('TOTAL_SLAVES') else int(
+                os.getenv('SLAVE_MUL', multiprocessing.cpu_count()))
+            # Default time duration to wait all slaves to be connected is 1 minutes / 60 seconds
+            SLAVES_WAITING_TIME = float(os.getenv('SLAVES_WAITING_TIME', 60))
+            # Default sleep time interval is 10 seconds
+            SLAVES_SLEEP_TIME = float(os.getenv('SLAVES_SLEEP_INTERVAL', 10))
```

Comment:
Variables naming is an opinionated thing ;) but I'd suggest to change the names to SLAVES_CHECK_TIMEOUT and SLAVES_CHECK_INTERVAL for better naming consistency. Also in line 73 it would be better if python var name would match the env var name.

Reply:
Also, could you reduce it to 5 seconds, please?

Reply:
Done